Client/Frontend/Widgets/ThumbnailCell.swift

```diff
@@ -180,6 +184,24 @@ class ThumbnailCell: UICollectionViewCell {
         textLabel.setContentCompressionResistancePriority(1000, forAxis: UILayoutConstraintAxis.Vertical)
     }
 
+    override func updateConstraints() {
+        super.updateConstraints()
+
+        imageWrapper.snp_remakeConstraints { make in
+            make.top.equalTo(self.contentView).inset(ThumbnailCellUX.Insets.top)
+            make.left.equalTo(self.contentView).inset(ThumbnailCellUX.Insets.left)
+            make.bottom.equalTo(self.contentView).inset(ThumbnailCellUX.Insets.bottom)
+            make.right.equalTo(self.contentView).inset(ThumbnailCellUX.Insets.right)
```

Comment:
I think this can be reduced to `make.edges.equalTo(self.contentView).inset(ThumbnailCellUX.Insets)`